jenkins/Jenkinsfile

```diff
@@ -59,9 +59,10 @@ pipeline {
                 vEnv()
                 sh script: '''
                     . .venv/bin/activate
-                  pip install -Ur rest-service/dev-requirements.txt
-                  pip install -Ur rest-service/test-requirements.txt
-                  pip install -e rest-service/
+                  cd rest-service
```

Comment:
Does this change have much benefit?

Reply:
i want to call the actual workflow in the restservice tests, so i have to install it

the alternative is copying the implementation, which seems worse. This way, the workflow is also tested.

Reply:
You're still pip installing the same code though, or am I misreading?

Reply:
Actually, this is neat enough anyway, not sure why I'm commenting- I'll mark this resolved.

Reply:
the contents of reqs.txt are going to be relative to cwd, and i put `../workflows/` inside of reqs.txt, and not `rest-service/workflows/`

i've done it this way, because we already do it this way in amqp-postgres